astropy/io/misc/hdf5.py

```diff
@@ -5,13 +5,13 @@
 `astropy.table`. See :ref:`table_io` for more details.
 """
 
-
 import os
 import warnings
 
 import numpy as np
+
+# DEV NOTE: Do not import anything from astropy.table here.
```

Comment:
Could you leave a ref to the issue?

Reply:
Never seen `DEV NOTE:` before. Any reason it's not a normal `NOTE:`?

Just wondering because neither PyCharm nor Spyder visually highlight this.

Reply:
Agree it should just be ``NOTE``

Reply:
Thanks for checking! I made up "DEV NOTE" (as this only affects developers who want to add code to this file). I'll address these comments shortly.